src/Optimizely/OptimizelyConfig/OptimizelyConfigService.php

```diff
@@ -223,11 +402,21 @@ protected function getExperimentsMaps()
         foreach ($this->experiments as $exp) {
             $expId = $exp->getId();
             $expKey = $exp->getKey();
+            $audiences = '';
+            if (is_null($exp->getAudienceConditions())) {
```

Comment:
instead of is_null use === null to avoid an extra function call

Reply:
No need to put this condition audiences is already empty str. Just put the else logic in an if block